config/flags.go

```diff
@@ -30,8 +30,9 @@ const (
 )
 
 const (
-	ListenAddrFlagName = "addr"
-	PortFlagName       = "port"
+	ListenAddrFlagName            = "addr"
+	PortFlagName                  = "port"
+	AdminEndpointsEnabledFlagName = "admin-endpoints-enabled"
```

Comment:
probably not worth but curious if there's any value in putting the admin API on a same port space to allow operators to better manage network security ingress by setting per port policies

Reply:
> on a same port space

Do you mean put it on a *different* port?

I think changes in this vein could be worth considering in the future, if driven by customer requirements.

Reply:
makes sense - feel free to resolve